### PR TITLE
Nuclear operatives have a christmas tree during the festive season

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Nuclear operatives have a Christmas tree during the festive season.
/:cl:

Minor buff to nuke ops during the Christmas period when trees have presents,
as they'll have access to random items just like the crew.

Before:
image

After:
image

### CODE REVIEW

**Comment by Nickkoranda** [comment]
and they say nukies are just made of salt

File: code/modules/events/holiday/xmas.dm [hunk 1]
```diff
@@ -46,12 +46,20 @@
 	var/christmas_tree = /obj/structure/flora/tree/pine/xmas/presents
 
 /obj/effect/landmark/xmastree/Initialize(mapload)
-	..()
+	. = ..()
+	// Xmas tree landmarks on Centcom will initialize BEFORE SSevent is initialized
```
Comment (optimumtact) [inline code/modules/events/holiday/xmas.dm]: This definitely doesn't make much sense, all atoms should call initialize at the same time when the atoms subsystem starts up, and then after that they call it immediately after new.

Comment (optimumtact) [reply code/modules/events/holiday/xmas.dm]: Nevermind, they're marked as INITALIZE IMMEDIATE

Comment (Cyberboss) [reply code/modules/events/holiday/xmas.dm]: landmerks are dumb

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**